.pre-commit-config.yaml

```
@@ -44,3 +44,9 @@ repos:
     - id: black-jupyter
       language_version: python3.8
       types: [jupyter]
+
+-   repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.5.1
+    hooks:
+    - id: prettier
+      types_or: [json]
```

Then we can remove the `.prettierrc` file
```suggestion
      types_or: [json]
      args: [--tab-width=2]
```

---

Ah that what it was!